conda_smithy/configure_feedstock.py

```diff
@@ -597,7 +603,6 @@ def _render_ci_provider(
                 )
                 forge_config[plat_arch]["enabled"] = True
 
-                fancy_platforms.append(fancy_name[platform])
```

Comment:
Why?

Reply:
Probably accidental.  I'll check of we still need it, we might not anymore